### PR TITLE
fix: fix licenses plugin licenses

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**
fix the licenses of the plugins files

**Motivation for the change:**
ensure that we are respecting the project license properly
ps: to do this pr i compare all files for we be sure over what is from sdk, modified from kb or fully kb

### CODE REVIEW

**Comment by jmrodri** [comment]
Are the changes in here correct? because I see the following was added to the license headers. 

```
Modifications copyright 2020 The Operator-SDK Authors
```

**Comment by camilamacedo86** [comment]
Hi @jmrodri, 

We have 3 scenarios : 
- SDK license when it was done by us
- Kube license when it was done by them and just copy and still 100% equals
- Kube license with modification statement when it was copied but is no longer equals the original file. 
Regards the modification one it cames from https://softwareengineering.stackexchange.com/questions/220068/file-with-apache-2-0-and-my-modifications

**Comment by jmrodri** [comment]
> Are the changes in here correct? because I see the following was added to the license headers.
> 
> ```
> Modifications copyright 2020 The Operator-SDK Authors
> ```

After speaking with @camilamacedo86 these changes are only to the files we created or re-wrote. The files that were copied still contain the correct copyright headers.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,23 +0,0 @@
-<!--
```
Comment (asmacdo) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Was this removed deliberately?

Comment (camilamacedo86) [reply .github/PULL_REQUEST_TEMPLATE.md]: good catcher .. really tks .. fixed.

**Pull request merged**